eosio.system/include/eosio.system/native.hpp

```diff
@@ -61,6 +61,14 @@ namespace eosiosystem {
    };
 
 
+   struct abi_hash {
```

Comment:
This isn't really part of the native APIs / actions / intrinsics. It is more appropriate to put this in eosio.system.hpp than native.hpp in my opinion.